core/node/rules/can_create_stream.go

```diff
@@ -233,6 +233,7 @@ func (ru *csParams) canCreateStream() ruleBuilderCS {
 				ru.checkMediaInceptionPayload,
 			).
 			requireMembership(
+				inception.UserId,
 				inception.ChannelId,
 				inception.SpaceId,
 			).
```

Comment:
you're right, UserId is not a stream, we want to use `requireUserAddr(inception.UserId)` instead, though `requireUserAddr` does not have a check for nil, you'll need to add that if you don't want to break the other stream types.